manifests/server/default_privileges.pp

```diff
@@ -62,15 +62,15 @@
   #
   # Port, order of precedence: $port parameter, $connect_settings[PGPORT], $postgresql::server::port
   #
-  if $port != undef {
+  if $port {
     $port_override = $port
-  } elsif $connect_settings != undef and 'PGPORT' in $connect_settings {
+  } elsif 'PGPORT' in $connect_settings {
     $port_override = undef
   } else {
     $port_override = $postgresql::server::port
   }
```

Comment:
the whole block doesn't make sense? Port is always set and cannot be undef. so `$port_override` is always set to `$port`? I don't thinky anybody uses `$connect_settings`. The whole setup seems broken?